google-cloud-storage/pom.xml

```diff
@@ -165,11 +165,6 @@
       <artifactId>google-cloud-conformance-tests</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Isn’t this dependency used in the file below?
https://github.com/googleapis/java-storage/blob/4ab4af704ead2331d4514a20c980cf878fcee858/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java#L120

Reply:
I assume now this dependency declaration should come back?

Reply:
No, it shouldn't. It caused problems here so it needs to move to the only pom.xml where it's used.